styles.css

```diff
@@ -656,4 +766,62 @@ header.active .navigations {
   .in-touch {
     margin-left: 75px;
   }
+
+  .pop {
+    display: none;
+    position: fixed;
+    justify-content: center;
+    align-items: center;
+    margin: 0 auto;
+    top: 0;
+    bottom: 0;
+    max-height: 100vh;
+    width: 100%;
+    overflow: auto;
+    background: rgb(193 199 208 / 50%);
+    backdrop-filter: blur(3px);
+    padding: 50px 16px;
+  }
+
+  .pop-content {
+    padding: 12px;
+    margin: 0 95px;
+    display: flex;
+    flex-direction: column;
+    justify-items: center;
+    width: 80%;
+    max-height: 100vh;
+    background: #fff;
+    border: 1px solid #ebecf0;
+    box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+    border-radius: 16px;
+  }
+
+  .popimg {
+    width: 100%;
+    height: 300px;
+    margin-bottom: 20px;
+  }
```

Comment:
- [ ] You have done a great job to ensure that the image is displayed as required by the previous reviewer, however since we have to match the Figma design, I suggest to consider making the image more responsive. In some desktop screen sizes, the image stretches and hence does not match the Figma design. Kindly ensure to apply this.
Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/233769878-017375a4-c499-4573-8ef0-39aa5f8834eb.png)

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/233769918-908d8652-dc41-485b-9884-8d5b7e4a0c34.png)